script/hassfest/manifest.py

```diff
@@ -117,19 +117,26 @@ class NonScaledQualityScaleTiers(StrEnum):
 ]
 
 
-def documentation_url(value: str) -> str:
+def core_documentation_url(value: str) -> str:
     """Validate that a documentation url has the correct path and domain."""
     if value in DOCUMENTATION_URL_EXCEPTIONS:
         return value
+    if not value.startswith("https://www.home-assistant.io/integrations"):
```

Comment:
Could we use a constant?
```suggestion
    if not value.startswith(_CORE_DOCUMENTATION_BASE):
```